components/MW/ConvertedGeneID/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { MetaNode } from '@/spec/metanode'
+import { z } from 'zod'
+import { VALID_LOADERS } from 'next/dist/shared/lib/image-config'
+
+export function uniqJsonSubset(data) {
+  let datastr = JSON.stringify(data);
+  datastr = datastr.replace("^[","");
+  datastr = datastr.replace("]$","");
+
+  const dataobj = JSON.parse(datastr);
```

Comment:
I'm not entirely certain what's going on here but if you're removing `^[` and `]$` from a json stringified data and re-parsing it then perhaps you're simply removing the outer list but this would only be valid if the you only have **one** list.. in which case shouldn't the code just be: `const dataobj = data[0]` ? or more elegantly `const [dataobj] = data`